parley/src/tests/test_basic.rs

```diff
@@ -59,7 +67,11 @@ fn only_inboxes_wrap() {
     }
     let mut layout = builder.build(text);
     layout.break_all_lines(Some(40.0));
-    layout.align(None, Alignment::Middle, AlignmentOptions::default());
+    layout.align(
+        layout.width(),
+        Alignment::Middle,
+        AlignmentOptions::default(),
+    );
```

Comment:
If we don't think `layout.width()` is a good default then we should probably also change the tests to use `40.0` here.

Reply:
Ditto the other tests that are line-broken with a specified width.

Reply:
Yes... you're probably right. I was a bit hesitant due to the snapshot churn it'd cause.